README.md

```diff
@@ -57,13 +58,29 @@ CORS can be configured in a `json` file that is pointed at from `package.json:ma
 
 ### bm-server.json
 
+By setting cors to `false`, Cross-Origin resource sharing will not be allowed. **This is the default behaviour**
+
+```json
+{
+  "cors": false
+}
+```
+
+By setting cors to `true`, defaults below will be used.
+
+```json
+{
+  "cors": true
```

Comment:
The main gotcha with the default origin "*" is that cookies and HTTP basic auth. won't work. Another concern is that it might be too permissive, and be slightly weaker from a security perspective.

I'd prefer it if there was no way for a customer to deploy something with an origin wildcard, but this is probably fine for now. We should ask our customers about this to see how they feel about the convenience-versus-security trade-off.